.azure-pipelines/pipelines.yml

```diff
@@ -94,16 +94,31 @@ jobs:
     dependsOn: ["format"]
     timeoutInMinutes: 360
     pool: "x64-large"
+    strategy:
+      maxParallel: 2
+      matrix:
+        coverage:
+          CI_TARGET: "bazel.coverage"
```

Comment:
make this CI_TARGET: "coverage"/"fuzz_coverage" then you can use it in upload step as well?

Reply:
Can you make this as displayName: "Linux-x64" in line 93? then the GH check name should match existing.